### DISCUSSION TITLE
Please help: Trying my best to set up WOL in Home app to turn on PC

### DISCUSSION THREAD

**Discussion by muchuk** [AlexGustafsson/homebridge-wol#228] [Questions]
Hi 

Please could someone kindly talk me through the set-up of WOL. I've read over the wiki and it doesn't solve the issue I'm encountering.

I've attached below the device config. I've taken out the IP and MAC addresses, but they're correctly entered.

The issue I have is with the Home App. When I go to add the accessory, I can see Wol Bridge listed as an available accessory. But when I select it, I get a pop-up asking for the Setup Code "enter the code found on the accessory, box or instructions."

The result is that I cannot add the accessory to the Home app to trigger the wake-up command.

All I want to achieve is turning on my PC via Siri/Home app. Nothing anymore complex.

   "accessories": [
 {
            "name": "PC",
            "manufacturer": "homebridge-wol",
            "model": "NetworkDevice",
            "host": "**IP ADDRESS"**, (obfuscated)
            "pingInterval": 2,
            "pingsToChange": 5,
            "pingTimeout": 1,
            "pingCommandTimeout": 0,
            "mac": "**MAC ADDRESS**",  (obfuscated)
            "startCommandTimeout": 0,
            "wakeGraceTime": 45,
            "wakeCommandTimeout": 0,
            "shutdownGraceTime": 15,
            "shutdownCommandTimeout": 0,
            "accessory": "NetworkDevice",
            "serialNumber": "225"

**Comment by AlexGustafsson** [comment]
So if I understand you correctly, you can't add Homebridge at all to your Home app? If so, check with the guys over at https://homebridge.io. There are links to GitHub, Discord and Reddit to the top right.

It's difficult to immediately point out a cause for this.  Have you added the bridge already? If so, there's no need to add the accessory as it should show up automatically.
